jsoncpp/PSPBUILD

```diff
@@ -9,6 +9,14 @@ makedepends=()
 source=("git+https://github.com/open-source-parsers/jsoncpp#commit=5defb4ed1a4293b8e2bf641e16b156fb9de498cc")
 sha256sums=('SKIP')
 
+prepare() {
+    cd "$pkgname/pkg-config"
+    sed -i 's#@CMAKE_INSTALL_PREFIX@#${PSPDEV}/psp#' jsoncpp.pc.in
+    sed -i 's#@CMAKE_INSTALL_PREFIX@#${prefix}#' jsoncpp.pc.in
```

Comment:
This line can be removed, it's not used.

Reply:
Thanks, got it